controllers/tekton/pipelinerun_controller_test.go

```diff
@@ -0,0 +1,91 @@
+// +build integration
```

Comment:
`// +build integration` was added here to imply that this is an integration test.